tools/purge_duplicate_rubyrep_triggers.rb

```diff
@@ -16,6 +16,9 @@
 GROUP BY relname
 SQL
 
+REGION = File.exists?("./REGION") ? File.read("./REGION").strip : "0"
```

Comment:
Since it is looking at the `REGION` file it looks like it can only be run from one side of the replication. Could you document where this should be run? (or if that is already documented then please ignore this)